README.md

```diff
@@ -89,6 +89,31 @@ How it looks like in my setup:
 
 <img src="docs/img/wemos_d1_installed.jpg" height="180px">
 
+# Using Owie
+
+## Parking your board (password protection)
+
+**WARNING:** Arming your board for parking **will** disable the emergency recovery mode (3 restarts), so if you forget your network password, the only way to recover is to reflash via USB.
+The normal OTA update mode will still be functional as normal (see below for OTA instructions).
+Disarming the board will restore the emergency recovery mode.
+
+Use these instructions if you want to be able to 'park' your onewheel using the power button sequence.
+The park functionality comes by interrupting all communication between the BMS and the controller, thus causing an error 16.
+This functionality can be removed quite easily by someone motivated enough and with enough knowledge; all that's required is to open up the board, remove Owie and solder the wires back together, or to reflash it via USB.
+
+1. Set an Owie network password in `Settings`.
+1. Tap the `Arm` button in `Settings`. This arms your board so you can put it into 'park'.
+1. When you need to park your board, just restart it. Always start from an on state then turn it off, then back on, then off again; all in under 10 seconds.
```

Comment:
I feel like the parking sequence must be higher in this paragraph. Maybe a high level overview even before warning, something like "When parking function is enabled, you can  immobilize your board by quickly power cycling the board twice. Once immobilized, you unlock the board by logging into Owie wifi, tapping a button and power cycling the board."

WDYT?